manage.py

```diff
@@ -3,7 +3,8 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "oh_data_uploader.settings")
```

Comment:
Please undo this one too. `manage.py` is created by `Django` right away and should remain untouched. Sorry for not noticing this one earlier!

Reply:
Yes, I'm still in the process of doing it. :sweat_smile: